src/satellite/jest.config.js

```diff
@@ -13,6 +13,6 @@ process.env = Object.assign(process.env, {
 module.exports = {
   ...baseConfig,
   rootDir: '../..',
-  testMatch: ['<rootDir>/src/satellite/test.js'],
+  testMatch: ['<rootDir>/src/satellite'],
```

Comment:
Running `pnpm test satellite` will give ten of these errors
```sh
No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
In C:\OSD\telescope
  387 files checked.
  testMatch: C:/OSD/telescope/src/satellite - 0 matches
  testPathIgnorePatterns: \\node_modules\\ - 387 matches
  testRegex:  - 0 matches
```

Having it originally with `testMatch: ['<rootDir>/src/satellite/test.js'],`
and then running `pnpm test satellite`
will still run the Satellite test suite 10 times